storage/local/storage.go

```diff
@@ -372,6 +379,16 @@ func (s *MemorySeriesStorage) Start() (err error) {
 
 	log.Info("Loading series map and head chunks...")
 	s.fpToSeries, s.numChunksToPersist, err = p.loadSeriesMapAndHeads()
+	openHeadCount := 0
+	for fp := range s.fpToSeries.fpIter() {
+		if series, ok := s.fpToSeries.get(fp); ok {
+			if !series.headChunkClosed {
+				openHeadCount++
```

Comment:
Changed to increment directly. 

Yeah we would avoid a second pass like this, but to your point this will be a blip during startup. The main reason I count here is to avoid crossing into the crash recovery code.